.gitignore

```diff
@@ -14,3 +14,4 @@
 /denoDist
 /npm
 /deno
+/npmEsmDist
```

Comment:
Can you please move it to the same place as in `.eslintignore`?
All ignore files should have the same order.